client-app/ui-kit/components/molecules/line-item/vc-line-item.vue

```diff
@@ -358,11 +358,11 @@ watchEffect(() => {
   &__slot {
     @apply flex-none empty:hidden;
 
-    &:has(.vc-quantity, * .vc-quantity) {
+    &:has(.vc-add-to-cart, * .vc-add-to-cart) {
       @apply w-[6.5rem];
     }
 
-    &:has(.vc-add-to-cart, * .vc-add-to-cart, .add-to-cart, * .add-to-cart) {
+    &:has(.add-to-cart, * .add-to-cart) {
```

Comment:
DO NOT remove vc-add-to-cart from here.
Add .vc-add-to-cart and NOT .vc-add-to-cart--hide-button